modules/data-model/forms/impl/src/main/java/io/uhndata/cards/forms/internal/serialize/filters/Creator.java

```diff
@@ -41,6 +43,17 @@
 @Component
 public class Creator extends BaseFilterFactory implements DataFilterFactory
 {
+    @Override
+    public List<SelectorDetails> getFilterDetails()
+    {
+        List<SelectorDetails> result = new ArrayList<>();
+        result.add(new SelectorDetails("createdBy",
+            "Only show results that were created by the specified user."));
+        result.add(new SelectorDetails("notCreatedBy",
+            "Only show results that were not created by the specified user."));
```

Comment:
"not" can easily go unnoticed (at least by me)
```suggestion
            "Only show results that were created by any user other than the specified user."));
```